src/locales/en_US.js

```diff
@@ -713,6 +716,15 @@ const strings = {
   err_token_not_in_wallet_title: 'Token not available',
   err_token_not_in_wallet_msg: 'The requested token is not supported or not enabled in your wallet.',
   err_no_address_title: 'No FIO Address found.',
+  fio_wallet_connect_add: 'Add',
+  fio_wallet_connect_remove: 'Remove',
+  fio_wallet_connect_return: 'Do not remove',
+  fio_show_connected_wallets: 'Show connected',
+  fio_connect_no_wallets: 'There are no wallets',
+  fio_connect_checkbox_text: 'I acknowledge that once connected, these wallet addresses become public',
```

Comment:
, one address from these wallets will become public

Reply:
@paullinator should it be `addresses from these wallets will become public` not `one address`?

Reply:
it will only be one address not all addresses so keep the string the same